src/app/docs/rich-text-display/rich-text-display-docs.component.html

```diff
@@ -0,0 +1,45 @@
+<sky-page>
+  <sky-docs-demo-page
+    moduleName="SkyRichTextDisplayModule"
+    moduleSourceCodePath="src/app/public/modules/rich-text-display/"
+    pageTitle="Rich text display"
+  >
+
+    <sky-docs-demo-page-summary>
+      <sky-alert alertType="warning">
+        This component is in beta. We do not plan to make any breaking changes to the component before the official release, but such changes are possible.
+      </sky-alert>
+      <p>
+        The rich text display component provides a safe way to display html strings with formatting on a web page.
+      </p>
+    </sky-docs-demo-page-summary>
+
+    <sky-docs-demo>
+      <sky-rich-text-display
+      [richText]="richText"
+      ></sky-rich-text-display>
+    </sky-docs-demo>
+
+    <sky-docs-code-examples
+      [packageDependencies]="{
+        '@blackbaud/auth-client': '*',
+        '@skyux/colorpicker': '*',
+        '@skyux/inline-form': '*',
+        '@skyux/list-builder': '*',
+        '@skyux/list-builder-common': '*',
+        '@skyux/list-builder-view-checklist': '*',
+        '@skyux/lists': '*',
+        '@skyux/lookup': '*',
+        '@skyux/popovers': '*',
+        '@skyux/tabs': '*',
+        '@skyux/text-editor': '*',
+        '@skyux/validation': '*',
+        'dompurify': '*'
+      }"
+    >
+      <sky-docs-code-example
+        sourceCodePath="src/app/public/plugin-resources/code-examples/rich-text-display/"
+      ></sky-docs-code-example>
+    </sky-docs-code-examples>
+  </sky-docs-demo-page>
+</sky-page>
```

Comment:
I don't see the JSDocs comment for the `richText` property, but if it's not too much of a pain, we should probably tweak that to follow our standard style for property descriptions and change `Rich text to display` to `Specifies the rich text to display.` Not a huge deal if we don't get to that though. ... Are there any other points that we should call out about this property though? Anything else at all that consumers should know about what it does or how to implement it?

Reply:
Done.
No, there's really not more to it. It's a simple input. There are no options or configuration that the consumer has to worry about.